tensorflow_datasets/image/structseg.py

```diff
@@ -0,0 +1,138 @@
+"""
+Organ-at-risk segmentation dataset from head & neck CT scans
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import re
+import tensorflow_datasets.public_api as tfds
+import tensorflow as tf
+import nibabel as nib
+import numpy
+
+_CITATION = """
+@misc{li_zhou_deng_chen, title={StructSeg2019 - Grand Challenge}, 
+url={https://structseg2019.grand-challenge.org/}, 
+journal={Automatic Structure Segmentation for Radiotherapy Planning Challenge }, 
+author={Li, Hongsheng and Zhou, Jinghao and Deng, Jincheng and Chen, Ming}}
+"""
+
+_DESCRIPTION = """
+The dataset is used for evaluating automatic algorithms on segmentation of organ-at-risk(OAR)
+of nasopharynx cancer, for radiation therapy planning. The dataset contains organ-at-risk 
+segmentation from head & neck CT scans 
+"""
+
+_BASE_URL = """https://structseg2019.grand-challenge.org/"""
+
+
+class Structseg(tfds.core.GeneratorBasedBuilder):
+    """
+    Organ-at-risk segmentation dataset from head & neck CT scans
```

Comment:
Should provide different builder configs for the different tasks since structseg has multiple.